main.go

```diff
@@ -1,32 +1,24 @@
 package main
 
 import (
-	"fmt"
 	"log"
-
-	_ "github.com/lib/pq"
-	"gorm.io/driver/postgres"
-	"gorm.io/gorm"
+	"os"
+
+	"github.com/joho/godotenv"
+	"github.com/mkrs2404/eKYC/api/services"
+	"github.com/mkrs2404/eKYC/database"
+	"github.com/mkrs2404/eKYC/server"
+	_ "github.com/mkrs2404/eKYC/server"
+	"gorm.io/gorm/logger"
 )
 
 func main() {
-
-	connection_string := "host=localhost user=postgres password=postgres port=5432 sslmode=disable"
-	db, err := gorm.Open(postgres.New(postgres.Config{DSN: connection_string}), &gorm.Config{})
-	handleError(err)
-
-	database, err := db.DB()
-	handleError(err)
-
-	err = database.Ping()
-	handleError(err)
-
-	fmt.Println("Connected to Postgres")
-
-}
-
-func handleError(err error) {
+	err := godotenv.Load()
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal("Error fetching the environment values")
+	} else {
+		database.Connect(os.Getenv("DB_HOST"), os.Getenv("DB_NAME"), os.Getenv("DB_USER"), os.Getenv("DB_PASSWORD"), os.Getenv("DB_PORT"), logger.Error)
```

Comment:
Try reading from command line args. Why?
- Developers are forced to pass at runtime. Credentials will not be stored in repo.

Reply:
For test, should I keep the env file?